README.md

```diff
@@ -196,6 +196,68 @@ environment variables:
 * `from_email` - The From address for notification emails
 * `force_result` - If set to `1` tickets in the [tracker openqa-force-result](https://progress.opensuse.org/projects/openqav3/issues?query_id=700) can override job results
 
+## Auto submission scripts for `devel:openQA`
+
+There are three scripts/pipelines responsible for managing submissions in OBS:
+* Trigger: `trigger-openqa_in_openqa`
+* Monitor: `monitor-openqa_job`
+* Submit: `os-autoinst-obs-auto-submit` / `cleanup-obs-project`
+
+They are called in [Jenkins](http://jenkins.qa.suse.de/)
+
+There are three projects in OBS:
+* [devel:openQA](https://build.opensuse.org/project/show/devel:openQA)
+* [devel:openQA:testing](https://build.opensuse.org/project/show/devel:openQA:testing)
+* [devel:openQA:tested](https://build.opensuse.org/project/show/devel:openQA:tested)
+
+[devel:openQA](https://build.opensuse.org/project/show/devel:openQA) is the
+project where the packages are automatically updated with every commit in git.
+
```

Comment:
Maybe an explanation of `devel:openQA:testing` here in the middle would be good:

```suggestion

[devel:openQA:testing](https://build.opensuse.org/project/show/devel:openQA:testing) contains
packages currently being tested for submission to `openSUSE:Factory` and potentially other targets
(via `devel:openQA:tested`, see next paragraph). It is only updated by the submission scripts themselves
to remain otherwise unchanged while tests are running and a submission is prepared. (If we would
submit directly from `devel:openQA`  we might submit packages that have already changed since tests
were executed so we would not necessarily submit what we have tested.)

```

Reply:
Something like that would be also nice 👍